src/app.ts

```diff
@@ -262,6 +309,109 @@ const removeProfile = async (name?: string): Promise<void> => {
   }
 };
 
+const removeAssumeRoleProfile = async (name?: string): Promise<void> => {
+  let profileName: string;
+
+  const assumeRoleProfiles = getAssumeRoleProfiles();
+
+  if (name) {
+    profileName = name;
+  } else {
+    const answers = await inquirer.prompt([
+      {
+        type: 'list',
+        name: 'profile',
+        message: 'Choose a profile',
+        choices: assumeRoleProfiles.map(profile => profile.profileName.replace('profile ', '')),
+        default: currentProfile || assumeRoleProfiles[0]
+      }
+    ]);
+
+    profileName = answers.profile;
+  }
+
+  const selectedProfile = getAssumeRoleProfile(profileName);
+
+  if (!selectedProfile) {
+    console.error(chalk.red(`No assumed role profile '${profileName}' found.`));
+
+    return;
+  }
+
+  const confirmAnswer = await inquirer.prompt([
+    {
+      type: 'confirm',
+      name: 'confirm',
+      message: `Are you sure you want to remove assumed role profile '${profileName}'?`
+    }
+  ]);
+
+  if (profileName && confirmAnswer.confirm) {
+    deleteAssumeRoleProfile(profileName);
+    console.log(chalk.green(`Removed assumed role profile '${profileName}'`));
+  }
+};
+
+const addAssumeRoleProfile = async (
+  name?: string,
+  parentProfile?: string,
+  roleArn?: string,
+  defaultRegion?: string,
+  outputFormat?: string
+): Promise<void> => {
+  if (name && parentProfile && roleArn && defaultRegion && outputFormat) {
+    const assumeRoleProfile: AssumeRoleProfileConfiguration = {
+      profileName: name,
+      parentProfileName: parentProfile,
+      roleArn,
+      defaultRegion,
+      outputFormat
+    };
+
+    createAssumeRoleProfile(assumeRoleProfile);
+  } else {
+    const profileAnswers = await inquirer.prompt([
+      {
+        type: 'input',
+        name: 'profile',
+        message: 'Name'
+      },
+      {
+        type: 'input',
```

Comment:
This could be a list input so they could just select one of their existing profiles.

Reply:
Good call